boosting_bbvi_tutorial.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
Can't give line comments in GitHub for notebooks unfortunately. Simply putting all comments here.

In the section, "The Greedy Algorithm," I would prefer if the extra stuff (imports, etc) was put somewhere else and the focus was put on the core for-loop, e.g. `for step in range(n_steps):`, and whatever is the minimal amount of things necessary to have that make sense. The full code can be included at the very end of the tutorial.

Reply:
Thanks, for the review @gideonite ! 

That's a good point, I've factored out the core loop, and then put the entire implementation at the end.